### DISCUSSION TITLE
How to do Delta Deployments for MDAPI format

### DISCUSSION THREAD

**Discussion by Hirishi** [scolladon/sfdx-git-delta#877] [Q&A]
We are currently deploying the whole package and we are looking forward to use SGD Delta. Could someone help us with the yml structure for MDAPI formatting.

**Comment by scolladon** [comment]
Yes sure! I can help
What do you want to do? What do you need?

**Reply by Hirishi** [reply]
So currently we have the below yml for validating/deploying the whole package.

![image](https://github.com/scolladon/sfdx-git-delta/assets/32322519/b9885af1-64b8-4c20-9b0e-85d09ace18d1)

When need a yml structure in a way only the changes in the PR(commits made to it) needs to be pushed. We are getting commands only for source format . But for MDAPI format we don't see any, could you please help us on the same

**Reply by scolladon** [reply]
MDAPI format is like a "subset" of the source format.
The source format allow more flexibility in terms of metadata location and also in term of metadata decomposition.

It means you can still use the sfdx-git-delta plugin with the metadata API format.
It will be at your own risk because we have not tested it but it "should" work

**Reply by Hirishi** [reply]
Can you please help me with the command that I need to use or any sample one that you have come across for MDAPI format?

**Reply by scolladon** [reply]
Sure, you can use the plugin the same as with the source format.
Please have a look to the [README](https://github.com/scolladon/sfdx-git-delta#tldr) to know how to use it
You'll have to make sure the parameters are set properly.
In your case, in the context of a PR CI job, here is what I would try first:
- `--to`: I would use "HEAD" or the work item branch (feature/x, fix/x, etc)
- `--from`: I would use the merge target branch (main, develop, release/x, etc)
